spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/RunControllerHelperService.java

```diff
@@ -124,7 +123,7 @@ public JSONObject changePlatformType(HttpSession session, JSONObject json) {
         if (newPt != null) {
           log.info("STORED: " + newRuntype + " :: " + storedPlatformType);
           if (!newRuntype.equals(storedPlatformType)) {
-            run = dataObjectFactory.getRunOfType(newPt, user);
+            run = new RunImpl(user);
```

Comment:
Difficult to say for sure since this makes a pass through the UI and we can't deploy it. Minimally, the platformtype needs set and a new SecurityProfile and Status should be set on the Run